src/components/Grid/Grid.test.tsx

```diff
@@ -99,4 +118,41 @@ describe("Grid", () => {
     cell && expect(cell.dataset.selected).toEqual("true");
     cell && expect(cell.dataset.focused).toEqual("true");
   });
+
+  it("should set row height to default (33px) when rowAutoHeight is false", async () => {
+    const { queryByTestId } = renderGrid({
+      rowCount: 10,
+      columnCount: 10,
+      rowAutoHeight: false,
+    });
+
+    const cell = queryByTestId("row-cell-0-0");
+    expect(cell).toBeDefined();
+
+    if (!cell) {
+      throw new Error("Cell with data-testid 'row-cell-0-0' not found");
+    }
```

Comment:
This can be removed. Line 130 will cause this test to fail if `cell` isn't defined.